tests/benchmark/run.py

```diff
@@ -25,6 +28,80 @@ def subtract(after_dict, before_dict):
     return {key: after_dict[key] - before_dict.get(key, 0) for key in after_dict}
 
 
+def prep_insert_statement(
+    df: pd.DataFrame, table_name="load_files_to_database", schema="Benchmark"
```

Comment:
What would happen when the user runs the same benchmarking script multiple times? `dag_id` would be same in that case as the row inserted before

Reply:
@sunank200 - I have logged an [issue](https://github.com/astronomer/astro-sdk/issues/460) for this bug and a few other bugs with the script.

Reply:
@sunank200 @utkarsharma2 We're adapting the benchmark to run as K8s jobs, as part of #432  so I believe the `dag_id` won't be an issue (each job will be run in a separate pod) - WDYT?